<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlgoLocalRunDialog</class>
 <widget class="QWidget" name="AlgoLocalRunDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>创建批号</string>
  </property>
  <widget class="QRadioButton" name="edgeRadio">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>边缘相机</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="faceRadio">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>50</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>面相机</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>451</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseFileBtn">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>50</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>执行</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
